src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx

```diff
@@ -14,70 +14,320 @@
+      </div>
+
+      <div className={styles.standardLabelValue}>
+        <div className={styles.label}>Transcript name</div>
+        <div className={styles.value}>{transcript.symbol}</div>
+      </div>
+
+      <div className={styles.standardLabelValue}>
+        <div className={styles.label}>Transcript length</div>
+        <div className={styles.value}>{transcript.slice.location.length}</div>
+      </div>
+
+      <div className={styles.standardLabelValue}>
+        <div className={styles.value}>
+          {transcript.spliced_exons.length} exons
+        </div>
+      </div>
+
+      <div className={`${styles.standardLabelValue} ${styles.spaceAbove}`}>
+        <div className={styles.label}>Coding exons</div>
+        <div className={styles.value}>{getNumberOfCodingExons(transcript)}</div>
+      </div>
+
+      <div className={styles.standardLabelValue}>
+        <div className={styles.label}>Combined exon length </div>
+        <div className={styles.value}>
+          {splicedRNALength} <span className={styles.unit}>bp</span>
+        </div>
+      </div>
+
+      {transcript.so_term === 'protein_coding' && (
+        <div className={styles.standardLabelValue}>
+          <div className={styles.label}>Protein</div>
+          <div className={styles.value}>
+            <div>
+              {productAminoAcidLength} <span className={styles.unit}>aa</span>
+            </div>
+            {product && (
+              <div className={styles.greyedText}>{product.stable_id}</div>
+            )}
+            {uniprotXref && (
+              <ExternalReference
+                classNames={{ label: styles.greyedText }}
+                label={'UniProtKB/Swiss-Prot'}
+                to={uniprotXref.url}
+                linkText={uniprotXref.accession_id}
+              />
+            )}
+          </div>
+        </div>
+      )}
+
+      {ccdsXref && (
+        <div className={`${styles.standardLabelValue} ${styles.spaceAbove}`}>
+          <div className={styles.value}>
+            <ExternalReference
+              classNames={{ label: styles.greyedText }}
+              label={'CCDS'}
+              to={ccdsXref.url}
+              linkText={ccdsXref.accession_id}
+            />
+          </div>
+        </div>
+      )}
+
+      <div className={`${styles.standardLabelValue} ${styles.spaceAbove}`}>
+        <div className={styles.value}>
+          <span
+            className={styles.downloadLink}
+            onClick={() => showDownload(!shouldShowDownload)}
+          >
+            Download
           </span>
+          {shouldShowDownload && (
+            <div className={styles.downloadWrapper}>
+              <InstantDownloadTranscript
+                genomeId={ensObjectGene.genome_id}
+                transcript={{
+                  id: transcript.unversioned_stable_id,
+                  so_term: transcript.so_term
+                }}
+                gene={{ id: gene.unversioned_stable_id }}
+                theme={'dark'}
```

Comment:
Whoa! Isn't it the opposite? Isn't this the light theme, like in your browser or your IDE that would be the light theme?

![image](https://user-images.githubusercontent.com/6834224/107395248-891ee580-6af4-11eb-8ef3-62847aa570ad.png)

Reply:
I am not sure. InstantDownload component only controls the colour of the text. When we ask it to give `dark` theme, it uses dark text.

Reply:
Swapped the theme as discussed